### PR TITLE
fix(@desktop): crash when transitioning to the app state

### LINKED ISSUE #3198: [Windows] Status is crashing when creating new account
Installing Status to C:/Status does not allow to create account, it crashes. Installing to AppData/Local does not make effect as well, it is closing.

Screenshot 2021-08-11 at 14 43 55

https://user-images.githubusercontent.com/82375995/129028168-974b70ad-9d44-4bf6-892d-f06aa956c5cf.mov

### PR DESCRIPTION
**Pull request by pinebit**
Fixes #3198

### CODE REVIEW

File: src/app/login/view.nim [hunk 1]
```diff
@@ -115,6 +115,8 @@ QtObject:
         msg = getCurrentExceptionMsg()
       result = StatusGoError(error: msg).toJson
 
+  proc moveToAppState*(self: LoginView) {.signal.}
+
   proc loginResponseChanged*(self: LoginView, error: string) {.signal.}
```
Comment (alaibe) [inline src/app/login/view.nim:120]: Do we still need this event?

**Review by alaibe — APPROVED** [review]

File: src/app/login/view.nim (hunk 1 above)
Comment (pinebit) [reply src/app/login/view.nim:120]: I did not check the code (but I should probably)...

Comment (pinebit) [reply src/app/login/view.nim:120]: discussed in Discord.

**Comment by anastasiyaig** [comment]
@pinebit i got this when logout from recovered account:

Screenshot 2021-08-11 at 17 26 49

**Pull request merged**